test/utils.d

```diff
@@ -85,6 +85,7 @@ auto payloadServer(scope HTTPServerRequest req, scope HTTPServerResponse res)
     APIExpectation expectation = void;
 
     // simple observer that checks whether a request is expected
+    synchronized {
```

Comment:
This is not needed to fix the weird regex error, but imho is a good idea to avoid further problems.